data/data/aws/bootstrap/main.tf

```diff
@@ -61,10 +61,12 @@ resource "aws_s3_bucket_object" "ignition" {
   server_side_encryption = "AES256"
 
   tags = merge(
+    {
+      "kubernetes.io/cluster/${var.cluster_id}" = "owned"
+    },
```

Comment:
Why add the cluster owned tag? I'm not sure if we consider this to be owned by the cluster and it looks like none of the other bootstrap resources are tagged similarly.